recirq/optimize/mgd.py

```diff
@@ -157,7 +157,7 @@ def model_gradient_descent(
         n_sample_points = int(
             np.ceil(n_sample_points_ratio * (n + 1) * (n + 2) / 2))
 
-    _, f = wrap_function(f, args)
+    _, f = oo._wrap_function(f, args)
```

Comment:
Let's get this fix in as a separate PR. Please see https://github.com/quantumlib/ReCirq/pull/184